### ISSUE TITLE
Contribuições.

### ISSUE THREAD

**Issue by dualvim** [filippovf/KitEasyFPGA#3]
Oi Filippo, excelente iniciativa. 
Tenho alguns scripts em VHDL aqui de alguns projetos com o ki Easy FPGA. Foram coisas que penei meses atrás e e somente encontrei esse  repo e seu blog hoje. 
Acredito que você já tenha scripts parecidos  conheça o assunto muito melhor que eu. Apenas sugeri enviar os scripts para contribuir com essa sua iniciativa, uma vez que estamos bem carentes de contribuições sobre esse tema aqui no Brasil

**Comment by filippovf** [comment]
Olá Eduardo,

Muito obrigado! Desculpe a demora, depois de um ano bem pesado eu desconectei de tudo o que estava ligado ao trabalho durante o recesso.

As contribuições são muito bem-vindas. Você quer agregá-las ao meu repositório, ou prefere criar um repositório seu e eu faço o link no meu?

Acho que foi uma placa com uma venda razoável por aqui e mundo afora (aliás, ainda vendem já que a Intel/Altera é bem generosa no tempo de suporte aos dispositivos) e o repositório visa contribuir mesmo. 

Um abraço,
Filippo

**Comment by dualvim** [comment]
Oi Filippo,
Eu posso ir agregando ao seu repositório sim. A melhor coisa é centralizar tudo em um só lugar e sua iniciativa foi excelente. s. Para mim o importante é reunirmos a nossa comunidade em um único local mesmo.

Se eu der um "fork" ai no seu repositório, as coisas que eu ir colocando no meu branch vão ficar facilmente visíveis pra você? Se for fica mais fácil para você dar as aprovações e ir atualizando fácil e também para atrair novos contribuintes.

Me desculpe a ignorância, mas nesses meus quase sete anos de Github eu praticamente só usei para subir coisas que já tinha finalizado  e já não me lembro mais como funciona. 

Um Abraço,

**Comment by filippovf** [comment]
Olá Eduardo,

Não seria com o Pull Requests? Vi que você fez o fork. Vamos ver se consigo jogar de volta pra cá. Eu também uso mais pra postar coisas individuais e observar/avaliar os alunos. No final como eles trabalham em grupo acabam ficando bem melhores que o professor, hehe.

Um abraço

**Comment by dualvim** [comment]
Oi Filippo,
Criei aqui um exemplo legal e enviei os arquivos através do meu Fork e dei um 'Pull Request' ai pra você. 
Vê ai se você recebeu meu Pull Request e me fala.  Se não tiver dado certo, eu tento outra vez.

Abraço,

**Comment by dualvim** [comment]
Aqui no meu repositório (https://github.com/dualvim/KitEasyFPGA/tree/master/Teste_Display7Segs) já está o exemplo que enviei. Acredito que para ir ai pro seu repo é só vc aceitar o meu Pull Request.

**Comment by marshel** [comment]
para registro, aqui tem alguns documentos que parecem ser oficiais da placa: http://fpga.redliquid.pl/

**Comment by dualvim** [comment]
Sim, esse é um dos poucos materiais disponíveis para o kit. O único porém é que o material usa versões beeem datadas do finado Quartus II e os exemplos que criam processadores do NIOS II usam o extinto "SOPC Builder". Para montar sistemas usando o NIOS II da Altera, agora é utilizado o Qsys (até a versão 16 ou 17 do Quartus) ou o Platform Designer (versões mais atuais do Quartus); 

Nevertheless, esse material continua sendo bom sim, pois reúne algumas coisas bastante importantes, como o diagrama com o esquema elétrico da placa e também contém dos documentos da Altera sobre os CIs da família Cyclone IV (buscar isso no site da Intel está péssimo e bem confuso. Melhor procurar via Google);

**Comment by dualvim** [comment]
Dei uma reestruturada nas pastas do meu repositório e adicionei alguns conteúdos interessantes. O repositório com o conteúdo novo é esse: https://github.com/dualvim/KitEasyFPGA/tree/Contribs_Dualvim . Também não esqueci de lhe dar os créditos Filippo
